### PR TITLE
VERSION: release runc v1.1.0

### PR DESCRIPTION
**Pull request by cyphar**
```
v1.1.0 -- "A plan depends as much upon execution as it does upon concept.

This release only contains very minor changes from v1.1.0-rc.1 and is
the first release of the 1.1.y release series of runc.

Changed:
 * libcontainer will now refuse to build without the nsenter package being
   correctly compiled (specifically this requires CGO to be enabled). This
   should avoid folks accidentally creating broken runc binaries (and
   incorrectly importing our internal libraries into their projects). (#3331)

Thanks to the following people who made this release possible:

 * Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp>
 * Aleksa Sarai <cyphar@cyphar.com>
 * Kir Kolyshkin <kolyshkin@gmail.com>
```

Signed-off-by: Aleksa Sarai <cyphar@cyphar.com>

### CODE REVIEW

**Review by AkihiroSuda — DISMISSED** [review]

**Review by AkihiroSuda — DISMISSED** [review]

**Comment by kolyshkin** [comment]
>  * runc can no longer be built with Go 1.15 or earlier. The minimum supported
>   Go version is now Go 1.16. (#3325)

Actually we do not support Go 1.15 since #3245 (which is in rc), and there is an item about in rc1 release notes:

> runc can only be built with Go 1.16 or later from this release onwards.
> (#3100, #3245)

So, #3325 was just another piece of Go 1.15 code removal. It doesn't hurt to say we no longer support 1.15 though. Perhaps this is fine as it is.

**Review by kolyshkin — DISMISSED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by cyphar** [comment]
@kolyshkin I'll fix the changelog in a follow-up PR (and I'll fix up the one used for the tag and release). Thanks.

**Comment by hqhq** [comment]
LGTM

**Review by AkihiroSuda — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -224,12 +233,16 @@ implementation (libcontainer) is *not* covered by this policy.
...
-[1.1.0-rc.1]: https://github.com/opencontainers/runc/compare/v1.0.0...v1.1.0-rc.1
+[Unreleased]: https://github.com/opencontainers/runc/compare/v1.1.0...HEAD
+[1.1.0]: https://github.com/opencontainers/runc/compare/v1.1.0-rc.1...v1.1.0
 [1.0.0]: https://github.com/opencontainers/runc/releases/tag/v1.0.0
 
-<!-- 1.0.x patch releases -->
+<!-- 1.0.z patch releases -->
 [Unreleased 1.0.z]: https://github.com/opencontainers/runc/compare/v1.0.3...release-1.0
-[1.0.3]: https://github.com/opencontainers/runc/compare/v1.0.2...v1.0.3
+[1.0.3]: https://github.com/opencontainers/runc/compare/v1.0.2..v1.0.3
```
Comment (AkihiroSuda) [inline CHANGELOG.md:242]: Nit:
```suggestion
[1.0.3]: https://github.com/opencontainers/runc/compare/v1.0.2...v1.0.3
```

Comment (cyphar) [reply CHANGELOG.md]: Fixed.

**Review by AkihiroSuda — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
still LGTM

**Pull request closed without merging**